app/importers/envirotech/relational_data.rb

```diff
@@ -0,0 +1,111 @@
+module Envirotech
+  class RelationalData
+    cattr_accessor :relations, :solution_ids_names
+
+    def import
+      return unless  RelationalData.relations.present?
+      process_solution_relations
+      process_regulation_relations
+    end
+
+    private
+
+    def process_regulation_relations
+      regulations = Envirotech::Consolidated.search_for(sources: 'regulations', size: 100)
```

Comment:
I am looking into your first suggestion, in the meantime, I fixed an issue with HTML escape characters, relational importing should be working correctly for now.
Also there was some data changes from the source, I updated the local data.